src/games/nogo_zestate.h

```diff
@@ -77,13 +77,16 @@ class StateForNogo : public State {
   virtual void ApplyAction(const _Action& action) override {
     assert(_status != GameStatus::player0Win);
     assert(_status != GameStatus::player1Win);
-    NoGoAction nogoAction(_nogoGame.GetTurnPlayer(), action.GetHash());
+    // NOTE: nhowe changed three occurrences of "GetHash" to "GetIndex",
+    //       which seems more correct. This was the only place that
+    //       "GetHash" was used.
+    NoGoAction nogoAction(_nogoGame.GetTurnPlayer(), action.GetIndex());
     if (_nogoGame.GetTurnPlayer() == PLAYER_0) {
-      _features[action.GetHash()] = 1.;
+      _features[action.GetIndex()] = 1.;
```

Comment:
Would be nice to make sure this didn't break anything

Reply:
Yeah IDK maybe this is what the state tests are for?